storybookjs/storybook
# Add loki to the docs

Issue: Loki is not in the docs, more people should know about it cause its handy.

## What I did
Not too sure where this should be in the docs, but with how great of a tool loki is I'm surprised it's not already.
## How to test

Is this testable with jest or storyshots? NR

Does this need a new example in the kitchen sink apps? ??

Does this need an update to the documentation? Y

If your answer is yes to any of these, please make sure to include it in your PR.
